packages/monaco-editor/just.config.ts

```diff
@@ -1,39 +1,15 @@
-import {
-  basicPreset,
-  task,
-  series,
-  parallel,
-  copyInstructions,
-  copyInstructionsTask,
-  cleanTask,
-  ts,
-  postprocess,
-  eslint,
-} from '@fluentui/scripts-tasks';
+import { preset, task, series, parallel, cleanTask, postprocessTask } from '@fluentui/scripts-tasks';
 
 import * as path from 'path';
 import { transformCssTask } from './tasks/transformCssTask';
 
-const monacoEditorPath = path.dirname(require.resolve('monaco-editor/package.json'));
```

Comment:
lots of duplication was removed - side effect of adding `lint-imports:all` to build post-step